src/test/java/uk/gov/ons/ctp/response/sample/Unirest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.ctp.response.sample;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import java.io.IOException;
+
+public class Unirest {
+
+    public static void initialiseUnirest() {
+        com.mashape.unirest.http.Unirest.setObjectMapper(new com.mashape.unirest.http.ObjectMapper() {
```

Comment:
import one object mapper

Reply:
I don't think we can, ones an interface and ones an implementation